keywords.txt

```diff
@@ -117,6 +117,8 @@ clearGdo2Action	KEYWORD2
 setTCXO	KEYWORD2
 setDio2AsRfSwitch	KEYWORD2
 getTimeOnAir	KEYWORD2
+implicitHeader
+explicitHeader
```

Comment:
The new keywords seem to be missing the type specifier, i.e.

impicitHeader<tab>KEYWORD2